common/controllers/form-wizard.js

```diff
@@ -30,6 +51,59 @@ class FormController extends Controller {
     next()
   }
 
+  post(req, res, next) {
+    // QUESTION: Not sure if this should live in `post`/`process`/somewhere else
```

Comment:
This seems like the best place to me - if you put anywhere later in the lifecycle such as `process` then it's going to get messy if another controller needs to override the method[0] - and the later it is, the more likely that's going to be.

[0] Unless the override stuff doesn't need to run until __after__ the existing super stuff